deploy/helm/charts/templates/deployment.yaml

```diff
@@ -122,6 +122,14 @@ spec:
 {{- end }}
 {{- if .Values.localpv.tolerations }}
       tolerations:
+        - effect: NoExecute
```

Comment:
can't we just set the `.Values.localpv.tolerations` value in the chart?
Otherwise this means you can never modify these, and `.Values.localpv.tolerations` become in addition to this

Reply:
But what if someone doesn't want them? Mayastor is just a use-case, we can make any requirements there without forcing it on every dynamic user
What if someone wants different seconds for example?

Reply:
I'll move it back to `.Values.localpv.tolerations`.

Reply:
Done. PTAL folks.

Reply:
I'm going to make a different suggestion.

On mayastor we have a separate field `earlyEvictionTolerations` which can be separately enabled/disabled with the "general" `tolerations`.
Would this achieve on we wanted?
1. It's always enabled by default, even if users sets other `tolerations`
2. Can be disabled if user doesn't want them by clearing `earlyEvictionTolerations`

What do you think?

Reply:
Sounds good.

Reply:
Alright, yeah that sounds good in this context.